httomo/runner/task_runner.py

```diff
@@ -332,6 +332,10 @@ def determine_max_slices(self, section: Section, slicing_dim: int):
 
         max_slices_methods = [max_slices] * len(section)
 
+        # NOTE: as the convertion of the raw data from uint16 to float32 happens after the data gets loaded,
+        # we should consider self.source.dtype to be float for memory estimators. This should change later
+        # when we deal with a possibility of data blocks being of different data type.
```

Comment:
I think it's fine to explain why something is the way it is in a code comment, but I think that anything about what will change in the future is best put into an issue and then referenced in the code comment.

So I think the first part is reasonable to have:
```python
        # NOTE: as the convertion of the raw data from uint16 to float32 happens after the data gets loaded,
        # we should consider self.source.dtype to be float for memory estimators.
```

but I think the next part:
```
This should change later when we deal with a possibility of data blocks being of different data type.
```

shouldn't just be in the source code, I think it's best put in an issue, and then referenced in the code comment.

This is so then work that is to be done in the future isn't documented solely in the source code, but in a github issue or something more easily "visible".